### PR TITLE
#11422: Replace tt_lib in models/experimental/t5,vovnet

### PR DESCRIPTION
**Pull request by mouliraj-mcw**
Issue : https://github.com/tenstorrent/tt-metal/issues/11422 Tracking Issue : https://github.com/tenstorrent/tt-metal/issues/11240 , Master Issue : https://github.com/tenstorrent/tt-metal/issues/10532
### Work Done:

- Replace tt_lib usage in `models/experimental/t5`
-  Replace tt_lib usage in `models/experimental/vovnet`

### Checklist
- [x] All Post commit CI 
- [ ]  Nightly Fast Dispatch Test
- [ ] Model regression CI testing passes

### CODE REVIEW

File: models/experimental/t5/tests/test_t5_block.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 import pytest
 import torch
 import json
-import tt_lib
+import ttnn
```
Comment (Aswinmcw) [inline models/experimental/t5/tests/test_t5_block.py:8]: Is ttnn import needed here?
Unused imports can be removed

File: models/experimental/t5/tt/t5_attention.py [hunk 2]
```diff
@@ -26,7 +25,7 @@ def t5_shape_tt(states, batch_size, n_heads, key_value_proj_dim, device):
         states = states.transpose(1, 2)
         tt_out = torch2tt_tensor(states, device)
     else:
-        tt_out = tt_lib.tensor.reshape(states, batch_size, -1, n_heads, key_value_proj_dim)
+        tt_out = ttnn.reshape(states, batch_size, -1, n_heads, key_value_proj_dim)
```
Comment (mcw-anasuya) [inline models/experimental/t5/tt/t5_attention.py:28]: Change this to ttnn.experimental.tensor.reshape

File: models/experimental/t5/tt/t5_attention.py [hunk 3]
```diff
@@ -57,7 +56,7 @@ def t5_unshape_tt(states, batch_size, inner_dim, device):
         tt_out = torch2tt_tensor(states, device)
     else:
         states = ttnn.transpose(states, 1, -2)
-        tt_out = tt_lib.tensor.reshape(states, 1, batch_size, -1, inner_dim)
+        tt_out = ttnn.reshape(states, 1, batch_size, -1, inner_dim)
```
Comment (mcw-anasuya) [inline models/experimental/t5/tt/t5_attention.py:59]: Same here

File: models/experimental/vovnet/tt/conv_norm_act.py [hunk 4]
```diff
@@ -9,7 +9,7 @@
 from models.utility_functions import torch_to_tt_tensor_rm
 from models.experimental.vovnet.vovnet_utils import create_batchnorm
 
-import tt_lib
+import ttnn
```
Comment (mcw-anasuya) [inline models/experimental/vovnet/tt/conv_norm_act.py:12]: Remove. Importing ttnn twice.

**Review by mcw-anasuya — APPROVED** [review]

**Review by eyonland — APPROVED** [review]

**Pull request merged**